packages/docusaurus-theme-classic/src/theme/NavbarItem/DocNavbarItem.tsx

```diff
@@ -52,7 +52,9 @@ export default function DocNavbarItem({
       {...props}
       className={clsx(props.className, {
         [activeSidebarClassName]:
-          activeDoc && activeDoc.sidebar === doc.sidebar,
```

Comment:
This doesn't seem to be necessary, because our custom `isActive` func determine if link is active or not:

https://github.com/facebook/docusaurus/blob/88de3a51065485148dbca70b2b0bca1e7237df40/packages/docusaurus-theme-classic/src/theme/NavbarItem/DefaultNavbarItem.tsx#L52-L55